### PR TITLE
Add self authorization fields to organizations and published sites reports

### LINKED ISSUE #4266: Add self authorization fields to Published Sites and Organizations reports
Include self authorization data in Published Sites and Organizations reports

## Notes

- The Organization model's self authorization data should be included in the Organizations and Published Sites reports (both in-browser and CSV versions)

## Acceptance Criteria

- [ ] Is Self Authorized column in Organizations report
- [ ] Is Self Authorized column in Organizations report CSV
- [ ] Is Self Authorized column in Published Sites report
- [ ] Is Self Authorized column in Published Sites report CSV

### PR DESCRIPTION
**Pull request by svenaas**
Fixes #4266

## Changes proposed in this pull request:
- Adds "Self Authorized" column to Organizations and Published Sites reports, containing data from the Organization model.
- Adds self authorization values to one org in the test data.

## security considerations
None. This change makes data already available to admins available in two reports available to admins.

### CODE REVIEW

**Review by drewbo — CHANGES_REQUESTED** [review]
`isSelfAuthorized` is a virtual field, so I think you can skip it in the `.create` method. Good to approve otherwise

File: admin-client/src/pages/organization/OrgsReport.svelte
Comment (apburnes) [inline admin-client/src/pages/organization/OrgsReport.svelte:27]: Do we care about a date just for self authorized? Should we just use the `isSelfAuthorized` boolean?

**Review by apburnes — COMMENTED** [review]
Just a quick question on the self auth data type. Other than that it looks good.

**Comment by svenaas** [comment]
> `isSelfAuthorized` is a virtual field, so I think you can skip it in the `.create` method. Good to approve otherwise

That's what I get for inspecting a model instance instead of the database or the code. Fixing it.

File: admin-client/src/pages/organization/OrgsReport.svelte
Comment (svenaas) [reply admin-client/src/pages/organization/OrgsReport.svelte:27]: Do we care about the date in general? Should this field just be stored as a boolean?

The reason I'm showing it is that I saw that it's shown in the admin orgs index page, so I thought I ought to be consistent and show it here. In the onscreen report I show boolean and date in one column, similar to how it's done in the orgs index page. 

In the CSV export if I were to only include one column in the CSV I would have thought it should be the date one, but right now I've got them in separate columns on the theory it might make downstream sort/filter semantics a small bit simpler.

Comment (apburnes) [reply admin-client/src/pages/organization/OrgsReport.svelte:27]: Reviewing it more, I think we only care if an org is or is not self authorized. Using `No` in conjunction with a date since for `Self Authorized` is mixing the types and the current Self Authorized field in the orgs table shows similar ambiguity with the timestamp and an `N/A`. If you want to change the org table's Self Authorized field to boolean too that would be great.

It should simplify the component and a future boolean filter on self authorization is more pertinent in targeting groups.

Comment (svenaas) [reply admin-client/src/pages/organization/OrgsReport.svelte:27]: All right, I will only show and export the boolean in these reports. I think changing the underlying field should happen on a separate issue and PR, though. I'll spin that up next.

**Review by apburnes — APPROVED** [review]
👍

**Comment by svenaas** [comment]
@drewbo, this is waiting for your approval too since you requested changes. It's not urgent.

**Review by drewbo — APPROVED** [review]

**Pull request merged**
